src/index.js

```diff
@@ -27,7 +36,10 @@ class HookActions {
  *          actions that can be invoked.
  */
 export function useGovernor(initialState = {}, contract = {}) {
-  if (!contract || typeof contract !== "object") {
+  if (
+    !contract ||
+    (typeof contract !== "object" && typeof contract !== "function")
+  ) {
     throw new TypeError(
       `contract is invalid: expected "object"; got "${typeof contract}"`,
```

Comment:
expected "object" or "class"?
or "class with instance methods"?

Maybe we even go full React on them and post a link to the documentation.